offer-manager-domain/src/main/groovy/life/qbic/business/offers/OfferItem.groovy

```diff
@@ -0,0 +1,63 @@
+package life.qbic.business.offers
+
+import groovy.transform.EqualsAndHashCode
+import life.qbic.business.offers.identifier.OfferId
+import life.qbic.datamodel.dtos.business.Affiliation
+import life.qbic.datamodel.dtos.business.Customer
+import life.qbic.datamodel.dtos.business.ProductItem
+import life.qbic.datamodel.dtos.business.ProjectManager
+import life.qbic.datamodel.dtos.projectmanagement.ProjectIdentifier
+
+/**
+ * <h1>Item that contains the information about a product item on the offer</h1>
+ *
+ * <p>This DTO should protect the business logic from being exposed to the external layers</p>
+ *
+ * @since 1.1.0
+ *
+*/
+@EqualsAndHashCode
+class OfferItem {
+    private double quantity
+    private String productDescription
+    private String productName
+    private double unitPrice
+    private String serviceProvider
+    private String unit
+    private double itemTotal
+
```

Comment:
I think the quantityDiscount is missing (we wanted to display it for every item afaik)